mlos_bench/mlos_bench/environments/local/local_env.py

```diff
@@ -191,6 +191,8 @@ def run(self) -> Tuple[Status, Optional[Dict[str, float]]]:
         if list(data.columns) == ["metric", "value"]:
             _LOG.info("Local results have (metric,value) header and %d rows: assume long format", len(data))
             data = pandas.DataFrame([data.value.to_list()], columns=data.metric.to_list())
+            # Try to convert string metrics to numbers.
+            data = data.apply(pandas.to_numeric, errors="ignore")   # type: ignore[assignment]  # (false positive)
```

Comment:
There's a test for this in the `test_ssh_env.py` expected output checks, but I can add something more explicit and split it out if you want too.

Reply:
Split out to #567